services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -208,14 +201,14 @@ const Asset = ({ libraryData }) => {
               <DashboardItem
                 aspectRatio={{ md: '2x1', lg: '16x9', xlg: '2x1' }}
                 border={['sm', 'md', 'lg', 'xlg']}
-                href="https://carbondesignsystem.com"
+                href={`${githubRepoUrl}/discussions/?discussions_q=in%3Atitle+${assetData.content.id}`}
               >
                 <dl>
-                  <dt className={dashboardStyles.label}>Coming soon...</dt>
+                  <dt className={dashboardStyles.label}>Discussions</dt>
                   <dd className={dashboardStyles.labelLarge}>–</dd>
                 </dl>
                 <Svg32Github className={dashboardStyles.positionBottomLeft} />
-                {pathIsAbsolute('https://carbondesignsystem.com') && (
+                {pathIsAbsolute(githubRepoUrl) && (
```

Comment:
side note to consider renaming this function to `isPathAbsolute` as a more conventional boolean function name.